ruby_on_rails/rails_basics/deployment.md

```diff
@@ -18,21 +18,21 @@ This section contains a general overview of topics that you will learn in this l
+You've already had some experience using a hosting provider when you deployed projects to Github Pages earlier in the curriculum. GitHub Pages is great for hosting **static** web pages for free, but we won't be able to use it for hosting our **dynamic** Ruby on Rails apps. We're going to need something more powerful.
 
 #### Static vs Dynamic Sites
 Static websites consist of pre-written HTML pages. They are "static" because everyone who visits them will see the same content. To build static sites, you only need HTML, CSS and Javascript.
 
 Dynamic websites, on the other hand, are websites that can change content based on the user who is visiting them. Twitter is a good example; every Twitter user sees different content on their homepage feed based on who they follow. To build dynamic sites, you still need HTML, CSS and JS. But additionally, you need a server-side language such as Ruby and a database.
 
-This additional tech prohibits us from using GitHub Pages for hosting our Ruby on Rails apps. GitHub pages cannot run Ruby code and doesn't have database services we can use.
+This additional tech prohibits us from using GitHub Pages for hosting our Ruby on Rails apps. GitHub Pages cannot run Ruby code and doesn't have database services we can use.
 
 Luckily, many hosting providers do offer everything we need. They range from the big and complex cloud providers like [AWS](https://aws.amazon.com/), [Google Cloud](https://cloud.google.com/) and [Microsoft Azure](https://azure.microsoft.com/) to the more beginner-friendly platform as a service (PaaS) providers like [Heroku](https://www.heroku.com/), [Railway](https://railway.app/) and [Fly.io](https://fly.io/). We will be focusing on and utilizing these latter providers in this lesson.
 
 ### What is a PaaS?
 Platform as a Service is a specific kind of hosting provider. The most important thing to know about them is they are much easier to use and more approachable for beginners than other hosting providers. They manage many of the low-level nitty-gritty details with the underlying server infrastructure. Allowing us as developers to focus more of our time on building our applications instead of configuring and managing the servers they run on.
 
-Taking the landlord metaphor, we used earlier a little further. A PaaS platform is like having a landlord who takes care of all the utilities, building maintenance and security. While you, the developer, focuses on furnishing, decorating and living in the space.
+Taking our landlord metaphor a little further, a PaaS platform is like having a landlord who takes care of all the utilities, building maintenance and security. While you, the developer, focus on furnishing, decorating and living in the space.
```

Comment:
Plural "focuses" reads better to me here.

Reply:
It's easier to see the subject-verb agreement if we take out "the developer":
"While you focus on furnishing …"

Reply:
I see what you mean, great idea!